openjdk/playlist.xml

```diff
@@ -86,7 +86,10 @@
 		<levels>
 			<level>extended</level>
 		</levels>
-		<!-- temporarily disable on openj9 due to https://github.com/eclipse/openj9/issues/10757 -->
+        <disabled>
+          <comment>https://github.com/eclipse/openj9/issues/10757</comment>
+          <impl>openj9</impl>
+        </disabled>
 		<impls>
 			<impl>hotspot</impl>
 		</impls>
```

Comment:
Because you are adding the `<disabled>` tag explicitly now, please remove this `<impls>` block as it is no longer the preferred way that we would exclude tests running on a particular platform.  You can remove from other `<test>`s where you have added the explicit `<disabled>` that contains `<impl>openj9</impl>` within it.